services/inference/tests/test_inference_lib.py

```diff
@@ -84,6 +85,7 @@ def _basic_result_checks(results: PredictOutput, df: pd.DataFrame):
     assert results["date"].iloc[-1] - df["date"].iloc[-1] == timedelta(hours=FORECAST_LENGTH)
 
 
+@pytest.mark.parametrize("forecasting_input_base", [TTM_MODEL_ID], indirect=True)
```

Comment:
Can we use this same test with "chronos"?

Reply:
assert for chronos is different. We can do asserts based on model. Separate methods would help if we use different parameters for both the models. We can even expand the `forecasting_input_base` parameters to consider model parameters.

Reply:
Extended all the TTM tests to Chronos.